packages/ckeditor5-modern/src/ModernEditor/ModernEditor.tsx

```diff
@@ -38,6 +42,9 @@ const ModernEditor: FC<ModernEditorProps> = ({
   } = editorConfig;
 
   const [files, setFiles] = useState<File[]>([]);
+  const [currentLoading, setUploadCurrentLoading] = useState<number>(0);
+  const [totaLoading, setUploadTotalLoading] = useState<number>(0);
+  const [isUploadLoading, setUploadLoading] = useState<boolean>(false);
```

Comment:
@nberongoy I think there's just too much variables here, you can probably remove `totalLoading` state?
As much as possible, let's not declare too much variables as it is going to cause confusion unless necessary.

Reply:
@vmc08 got it will direct the value of the total to the props and rename the variable to make it much more understanble